.github/workflows/release-plan-preview.yml

```diff
@@ -0,0 +1,50 @@
+name: Release Plan Review
+on:
+  push:
+    branches:
+      - main
+
+env:
+  VOLTA_FEATURE_PNPM: 1
+  branch: "gh-workflow/release-preview"
+  title: "Preview Release"
+
+jobs:
+  update_unstable_branch:
+    name: Update Unstable Branch
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    outputs:
+      explanation: ${{ steps.explanation.outputs.text }}
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/setup
+      - name: 'Prepare git branch'
+        run: |
+          set +e
+          git branch -D ${{ env.branch }}
+          set -e
+          git switch -c ${{ env.branch }}
+
+      - id: explanation
+        run: |
+          echo "text=$(pnpm embroider-release explain-plan)" >> $GITHUB_OUTPUT
+      - run: pnpm embroider-release gather-changes
```

Comment:
I don't think you need the `gather-changes` step. `prepare` will do the gather automatically if you aren't using `--stdin` to pass them in.

Reply:
thanks!!